docs/reference/faq.md

```diff
@@ -0,0 +1,190 @@
+# Known issues
```

Comment:
the file is named FAQ but then there's this section name, and also, the content is closer to a FAQ.

I wouldn't call it "Known issues". For issues, people should go to the GitHub issues page (in fact you can add a link to it here).

A FAQ is more useful. But is it normal to have a FAQ under Reference? Just asking... are there any other docs doing that?

Finally, I'd remove transient issues from this page, for your own benefit. You'll most likely not be able to keep this doc page up to date as fixes and feature requests get merged, and you'll end up giving readers the wrong info. I'll comment below the questions I believe are useful to keep, and the ones I feel are controversial

Reply:
I will change the title to FAQ.

> A FAQ is more useful. But is it normal to have a FAQ under Reference? Just asking... are there any other docs doing that?

@yhontyk do you know?

Reply:
Changed the title. Waiting on @yhontyk's input on the category.

Reply:
Pushed 2fff6cb per your comments @yhontyk.

Reply:
thanks